### PR TITLE
Blue Grid Land Surface

### PR DESCRIPTION
**Pull request by 733737**
I think this makes logical sense, RGB and RGBY are standards accepted everywhere... I need help with translations to foreign languages

### CODE REVIEW

File: objects/official/terrain_surface/official.terrain_surface.grid_blue/object.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	 { "path": "images/icon.png", "format": "raw", "x": 2, "y": 2 },
+	 "$G1[2257..2313]"
+    ],
+    "strings": {
+        "name": {
+            "en-GB": "Grid (Blue)",
+            "de-DE": "Gitter (Blue)",
+            "nl-NL": "Grid (Blue)",
+            "pt-BR": "Malha (Blue)",
+            "ja-JP": "グリッド (緑)",
```
Comment (Broxzier) [inline objects/official/terrain_surface/official.terrain_surface.grid_blue/object.json:24]: 緑 is green

File: objects/official/terrain_surface/official.terrain_surface.grid_blue/object.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            "fr-FR": "Grille (Blue)",
+            "pl-PL": "Krata (Blue)",
+            "ko-KR": "격자 (녹색)",
+            "eo-ZZ": "Krado (Blue)",
+            "sv-SE": "Rutnät (Blue)",
+            "ca-ES": "Graella (Blue)",
+            "fi-FI": "Ruudukko (Blue)",
+            "zh-CN": "网格(绿)"
```
Comment (Broxzier) [inline objects/official/terrain_surface/official.terrain_surface.grid_blue/object.json:32]: All except for the English translation are wrong.

File: objects/official/terrain_surface/official.terrain_surface.grid_blue/object.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    "properties": {
+        "colour": "light_blue",
+        "mapColours": [138, 10],
+        "price": 110
+    },
+    "images": 
+	[ 
+	 { "path": "images/icon.png", "format": "raw", "x": 2, "y": 2 },
+	 "$G1[2257..2313]"
+    ],
```
Comment (Broxzier) [inline objects/official/terrain_surface/official.terrain_surface.grid_blue/object.json:17]: Keep the style consistent:

```suggestion
    "images": [{
            "path": "images/icon.png",
            "format": "raw",
            "x": 2,
            "y": 2
        },
        "$G1[2257..2313]"
    ],
```

File: objects/official/terrain_surface/official.terrain_surface.grid_blue/object.json (hunk 2 above)
Comment (Broxzier) [reply objects/official/terrain_surface/official.terrain_surface.grid_blue/object.json:32]: Just leave them out for now.

**Comment by Gymnasiast** [comment]
Please do not keep reopening and closing pull requests, just update the existing one.

**Comment by 733737** [comment]
that was an error, i apologize

**Pull request closed without merging**